models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -1160,7 +1160,7 @@ FROM
     ('equad-quadrantprotocol', 'ethereum', 'EQUAD', 0xc28e931814725bbeb9e670676fabbcb694fe7df2, 18),
     ('fnk-fnkcom', 'ethereum', 'FNK', 0xb5fe099475d3030dde498c3bb6f3854f762a48ad, 18),
     ('fyz-fyooz', 'ethereum', 'FYZ', 0x6bff2fe249601ed0db3a87424a2e923118bb0312, 18),
-    ('gmt-gomining-token', 'ethereum', 'GMT', 0x7ddc52c4de30e94be3a6a0a2b259b2850f421989, 18),
+    ('gmt-stepn', 'ethereum', 'GMT', 0xe3c408bd53c31c085a1746af401a4042954ff740, 8),
```

Comment:
i don't think we should override this other token. they both seem valid to me. it's okay to share `GMT` symbol across multiple addresses. as long as address / blockchain combo is unique, we're good (and CI test will look for that)

let's leave the old one as-is, then add this as new

Reply:
Sure. What should I do?